app/scripts/controllers/preferences.js

```diff
@@ -178,17 +189,6 @@ export default class PreferencesController {
     this.store.updateState({ useCurrencyRateCheck: val });
   }
 
-  /**
-   * Setter for the `openSeaEnabled` property
-   *
-   * @param {boolean} openSeaEnabled - Whether or not the user prefers to use the OpenSea API for NFTs data.
-   */
-  setOpenSeaEnabled(openSeaEnabled) {
-    this.store.updateState({
-      openSeaEnabled,
-    });
-  }
-
```

Comment:
Is this change required?  It looks exactly the same -- if it's the same, I don't think we need the change, as it makes the PR larger than necessary.

Reply:
Oh yeah, this change is unnecessary now that I removed the clearKnownMethodData method